examples/shardingsphere-parser-example/src/main/java/org/apache/shardingsphere/example/parser/mysql/format/MySQLParserFormatExample.java

```diff
@@ -15,22 +15,37 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.example.parser;
+package org.apache.shardingsphere.example.parser.mysql.format;
 
 import org.antlr.v4.runtime.tree.ParseTree;
 import org.apache.shardingsphere.sql.parser.api.SQLParserEngine;
 import org.apache.shardingsphere.sql.parser.api.SQLVisitorEngine;
 
 import java.util.Properties;
 
-public class SQLParserFormatExample {
-
+public class MySQLParserFormatExample {
+    
+    private static final String DML_SELECT_SQL = "select age as b, name n from table1 join table2 where id=1 and name='lu';";
+    
+    private static final String DML_INSERT_SQL = "insert into user (name, age, status) values ('z', 18, 1);";
+    
+    private static final String DML_DELETE_SQL = "delete from user where id = 1;";
+    
+    private static final String DML_UPDATE_SQL = "update user set name = 'j' where id = 1;";
+    
+    private static final String DDL_CREATE_SQL = "create table user (id bigint(20) PRIMARY KEY AUTO_INCREMENT, name varchar(20), age int(2), status int(1));";
```

Comment:
@zjcnb Please keep all mysql keywords in uppercase.

Reply:
ok.